source/common/network/utility.cc

```diff
@@ -91,8 +91,7 @@ Address::InstanceConstSharedPtr Utility::resolveUrl(const std::string& url) {
   // TODO(mattklein123): We still support the legacy tcp:// and unix:// names. We should
   // support/parse ip:// and pipe:// as better names.
   if (url.find(TCP_SCHEME) == 0) {
```

Comment:
I think it's missing right now, but can we add a few explicit tests for this function in https://github.com/lyft/envoy/blob/master/test/common/network/utility_test.cc (to verify we get back the right address type on the returning address). Also, I think you can go ahead and delete the TODO at line 90 above, and also please delete the TODO at 91 since I don't think we are going to do that anymore.